Makefile

```diff
@@ -102,10 +102,12 @@ AMPBOOTIMG := appcelerator/$(AMP)-bootstrap:$(AMPTAG)
 AMPTARGET := $(CMDDIR)/$(AMP)/$(AMPBINARY)
 # TODO: add api/client back to project
 #AMPDIRS := api/client $(CMDDIR)/$(AMP) tests
-AMPDIRS := $(CMDDIR)/$(AMP) tests
+AMPDIRS := $(CMDDIR)/$(AMP) cli tests
 AMPSRC := $(shell find $(AMPDIRS) -type f -name '*.go')
 
 $(AMPTARGET): $(CMDDIR)/$(AMP)/Dockerfile $(GLIDETARGETS) $(PROTOTARGETS) $(AMPSRC) $(AMPBOOTDIR)/$(AMPBOOTEXE)
+	@echo "Compiling $(AMP) source(s):"
+	@echo $?
```

Comment:
I actually just recently added it because it *really* helps to confirm what's getting built while hacking on it. The long silent pause wasn't helping either, and unless we want to suppress all the docker output as well, it's not like these are actually silent commands right now. I'd like to keep these here for the time being. We can revisit it again in a couple of weeks.